aaad/validators.py

```diff
@@ -52,22 +55,24 @@ def validate_resource_quotas(self, act_as, action, request_body):
         resource_quotas = ""
         resource_quotas = utils.parse_permissions_file(quota_file).get(act_as, {})
         resource_quotas = resource_quotas.get("resources", {})
-        cpu_quota = float(resource_quotas.get("cpu", 0.0))
+        cpu_quota = float(resource_quotas.get("cpus", 0.0))
         mem_quota = float(resource_quotas.get("mem", 0.0))
         disk_quota = float(resource_quotas.get("disk", 0.0))
         requested_cpu = float(requested_resources.get("cpus", 0.0))
         requested_mem = float(requested_resources.get("mem", 0.0))
         requested_disk = float(requested_resources.get("disk", 0.0))
+        valid_quota_request = True
+        self.message = ""
```

Comment:
How about a list `self.message = []` and then `append` messages?
 That way the caller can decide what to do with multiple messages.